io/src/obj_io.cpp

```diff
@@ -390,35 +389,40 @@ pcl::OBJReader::readHeader (const std::string &file_name, pcl::PCLPointCloud2 &c
       sstream.imbue (std::locale::classic ());
       line = sstream.str ();
       boost::trim (line);
```

Comment:
Do we actually need this block? Below we only test first characters of the line, why messing with locale and trimming?

Reply:
That's true, I've removed the lines in e7fbf98, but the speed improvement seems to be marginal.